Demo/src/main/java/com/braintreepayments/demo/PayPalRequestFactory.java

```diff
@@ -90,6 +90,7 @@ public static PayPalCheckoutRequest createPayPalCheckoutRequest(
             shippingAddress.setLocality("San Francisco");
             shippingAddress.setRegion("CA");
             shippingAddress.setCountryCodeAlpha2("US");
+            shippingAddress.setPostalCode("94103");
```

Comment:
Just curious if the postal code is needed for something else

Reply:
It looks like it's dependent on the country code provided in the request. This might be new validation that the backend is doing for US 🤔